pass-fedora-client/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.dataconservancy.pass</groupId>
+    <artifactId>pass-client</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>pass-fedora-client</artifactId>
+  <name>pass-fedora-client</name>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <dependencies>
+	<dependency>
+      <groupId>org.dataconservancy.pass</groupId>
+      <artifactId>pass-model</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+	<dependency>
+      <groupId>org.dataconservancy.pass</groupId>
+      <artifactId>pass-json-adapter</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.dataconservancy.pass</groupId>
+      <artifactId>pass-client-util</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+	<dependency>
+	  <groupId>org.fcrepo.client</groupId>
+	  <artifactId>fcrepo-java-client</artifactId>
+	  <version>${fcrepo-java-client.version}</version>
+	</dependency>
+    
+    <dependency>
```

Comment:
I'd prefer to either eliminate the dependency on a logging impl, or include it as a test dependency.  I think `<optional>` works to exclude the dep from the runtime/compile-time classpath, but if it is required for tests, l'd just `<scope>` it to `test`, and leave it at that.  I think this goes for all modules (e.g.`pass-client-util`).  IMO, leave concrete logging impl choices nearest the user as possible.  For example, configuring it in an uber-jar (if this project ever needs one).  Otherwise, just configure concrete implementations as `test` dependencies.

Reply:
Also, we should be using `slf4j-api` as a compile time dependency for logging.